frontend/src/app/view-claims/view-claims.component.html

```diff
@@ -5,7 +5,7 @@
         <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
         <p [lyTyp]="'body1'">Vehicle Model: {{ claim.vehicle.model }}</p>
         <p [lyTyp]="'body1'">Vehicle Color: {{ claim.vehicle.color }}</p>
-        <p [lyTyp]="'body1'">Claim Status: {{ claim.vehicle.status }}</p>
+        <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
```

Comment:
![applause](https://i.giphy.com/media/l9Tllo1thElT5gvVOU/giphy.webp)

Thank god for you, @dbartalino.